### PR TITLE
The port that steamport was too afraid to port [READY FOR TESTMERGE]

### PR DESCRIPTION
**Pull request by ghost**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## About The Pull Request

Lazily cherry-picks https://github.com/tgstation/tgstation/pull/47749

That's pretty much it. I'll try my best to fix it and make sure nothing's broken, but there's so much shit here that I doubt it'll be a one-man job.

Also closes https://github.com/HippieStation/HippieStation/pull/12257

## Why It's Good For The Game

shit piss fuck

### CODE REVIEW

File: tgui-next/packages/tgui/public/tgui-main.html [hunk 1]
```diff
@@ -48,12 +67,50 @@
...
+<!-- Fatal error container -->
+<div id="FatalError" style="display: none">
+<div class="FatalError__logo">
+          _____            _____                  _______                  _____
+         /\    \          /\    \                /::\    \                /\    \
+        /::\____\        /::\    \              /::::\    \              /::\    \
+       /::::|   |        \:::\    \            /::::::\    \            /::::\    \
+      /:::::|   |         \:::\    \          /::::::::\    \          /::::::\    \
+     /::::::|   |          \:::\    \        /:::/~~\:::\    \        /:::/\:::\    \
+    /:::/|::|   |           \:::\    \      /:::/    \:::\    \      /:::/__\:::\    \
```
Comment (ghost) [inline tgui-next/packages/tgui/public/tgui-main.html:85]: fucking incredible

**Review by ghost — COMMENTED** [review]
I looked through every single file to make sure nothing hippie was affected. @yoyobatty , you probably need to look over this massive PR too

**Comment by ghost** [comment]
one other thing I forgot to mention: @stylemistake is an absolute legend who will be revered amongst the ss13 coders. There is no doubt in my mind about that.

**Comment by ghost** [comment]
jesus this is beautiful

File: tgui-next/packages/tgui/public/tgui-main.html (hunk 1 above)
Comment (stylemistake) [reply tgui-next/packages/tgui/public/tgui-main.html:85]: Glad you like it, it was fun making it.

**Comment by ghost** [comment]
I was waiting for other tgui-next PRs to be merged, so I could shove them all into one PR.

Now let me one-up you.

**Pull request closed without merging**